src/finch/julia.py

```diff
@@ -1,6 +1,13 @@
 import juliapkg
 
-juliapkg.add("Finch", "9177782c-1635-4eb9-9bfb-d9dfa25e6bce", version="0.6.20")
+deps = juliapkg.deps.load_cur_deps()
+if not (
+    "packages" in deps and
+    "Finch" in deps["packages"] and
+    deps["packages"]["Finch"]["version"] == "0.6.20"
```

Comment:
I'd simplify this to:
`deps.get(packages, {}).get("Finch", {}).get("version", None) == "0.6.20"`.

Reply:
Done!